.circleci/soltest.sh

```diff
@@ -75,6 +75,7 @@ do
         "--color_output=no"
         "--show_progress=yes"
         "--logger=JUNIT,error,test_results/$(get_logfile_basename "$((CPUs * CIRCLE_NODE_INDEX + run))").xml"
+        "--logger=HRF,error,stderr"
```

Comment:
Can you add this also in `soltest.ps1`? And do a test in CI by introducing some artificial error to make sure it behaves as expected on Windows.

Reply:
Yes, I will be doing that.